app/models/spree/variant_decorator.rb

```diff
@@ -0,0 +1,20 @@
+Spree::Variant.class_eval do
+  
+  alias_method :original_price_in, :price_in
```

Comment:
@rishabhguptarishi i don't think we need alias method if we simply use get_price method in place of **self.sale_price** in price_in method's logic. Lets discuss this if its not clear.